.idea/misc.xml

```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_1_8" project-jdk-name="1.8" project-jdk-type="JavaSDK">
-    <output url="file://$PROJECT_DIR$/build/classes" />
-  </component>
-  <component name="ProjectType">
-    <option name="id" value="Android" />
-  </component>
-</project>
```

Comment:
We don't need any change to `.idea` file, this must have been done automatically from android studio please do not add this file to your commit. 
Note: Don't delete this file, we need this file we just don't need the changes this file contains.